week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -57,8 +78,42 @@ function exerciseTwo(shopping) {
 
     The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com/
 **/
+
```

Comment:
I'm getting this error when I try to run this file

```
Uncaught SyntaxError: Unexpected end of input
```
So your code won't compile. Can you please fix this?

Reply:
i fixed it and please run again